src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Semantic/TestFiles/DefaultRazorSemanticTokenInfoServiceTest/ApplyEditsToPreviousCSharpDoc_EnsureAccurateDeletionEdits.semanticexpected.txt

```diff
@@ -1,13 +1,13 @@
 ﻿//line,characterPos,length,tokenType,modifier
 0 0 4 15 0 //keyword
-0 5 1 30 0 //method name
-0 1 1 36 0 //punctuation
```

Comment:
These were previously marked as `module name` according to the legend. How were these expectation files generated?